content/authentication/guides/systemauthentication-for-systemproviders/_index.nb.md

```
@@ -6,210 +6,206 @@ toc: false
-Eksempelet nedenfor viser et system som har behov for tilgang til tjenesten [Krav og betalinger](https://skatteetaten.github.io/api-dokumentasjon/api/kravogbetalinger) fra Skattedirektoratet som er [registrert i Altinn ressursregister](https://platform.tt02.altinn.no/resourceregistry/api/v1/resource/ske-krav-og-betalinger). 
 
-Senere vil Systembruker støtte tilgangspakker som er en samling av rettigheter på tvers av tjenester innfor et område.
+**Id:** Unik id som benyttes for å identifisere sluttbrukersystemet. Gyldige tegn er a-z 0-9 og _. Må være i form [orgnr]_[navn].  
+**Vendor:** Orgnr til levereansør 0192 er referanse til Enhetsregisteret i [Electronic Adress Scheme](https://docs.peppol.eu/poacc/billing/3.0/codelist/eas/)</br>
+**Name:** Navn på sluttbrukersystemet opprettelse. Angis på bokmål(nb), nynorsk(nn), engelsk(en))</br>
+**Beskrivelse:** Beskrivelse vises i altinn.no ved sluttbrukerstyrt opprettelse (bokmål, nynorsk, engelsk)</br>
+**Rights**: Nødvendige enkelttjenester for systembruker ift tjenestene den skal brukes mot. Referanser til applikasjoner i Altinn plattformen eller tjenester utenfor Altinn som er registrert hos Altinn.</br>
+**AccessPackages:** Nødvendige tilgangspakker for systembruker (Q2 2025).
+**AllowedRedirectUrls** Hviteklisting av gyldige url det kan rediretes til ved leverandørstyrt opprettelse (angis i RedirectUrl)</br>
+**ClientID:** Klientidene for integrasjonen som er opprettet i Maskinporten. 
+**IsVisible:** Angir om systemet skal vises systemregisteret for [Sluttbrukerstyrt opprettelse]().
```

markdownlint-cli2 (0.17.2)

No empty links
null

(MD042, no-empty-links)